includes/discovery/sensors/rittal-cmc-iii-sensors.inc.php

```diff
@@ -36,15 +38,34 @@
     $var_type = $entry['cmcIIIVarType'];
     $sensor_id = count($cmc_iii_sensors);
 
-    if ($cmc_iii_sensors[$sensor_id]['name'] != $sensor_name) {
+    $index_r = explode('.', $index);
+    if (count($index_r) > 1) {
+        $current_index_prefix = $index_r[0];
+    }
+
+    if ($cmc_iii_sensors[$sensor_id]['name'] != $sensor_name || $last_index_prefix != $current_index_prefix) {
```

Comment:
@HolgerHees 
I know the first part existed before and you didnt touch it, but..
```php
$sensor_id = count($cmc_iii_sensors);
...
if ($cmc_iii_sensors[$sensor_id] ...
```
This will false fail since it's outside of the array bounds. 
Are they trying to check the last array member?

Reply:
should be fixed in the new pull request